File: web/tests/reference/heaps/kth-largest-element-in-a-stream.py
class KthLargest:
    def __init__(self, k, nums):
        self.k = k
        self.values = sorted(nums)

    def add(self, val):
        self.values.append(val)
        self.values.sort()
        return self.values[len(self.values) - self.k]

File: web/tests/reference/heaps/kth-largest-element-in-a-stream.js
class KthLargest {
  constructor(k, nums) {
    this.k = k;
    this.values = [...nums].sort((a, b) => a - b);
  }

  add(val) {
    this.values.push(val);
    this.values.sort((a, b) => a - b);
    return this.values[this.values.length - this.k];
  }
}
